gulpfile.js

```diff
@@ -246,21 +246,21 @@ gulp.task('ext:test', (done) => {
 	var args = ['--verbose', '--disable-gpu', '--disable-telemetry', '--disable-updates', '-n'];
 	let vscodeVersion = packageJson.engines.vscode.slice(1);
 	let extensionTestsPath = `${workspace}/out/test`;
-	vscodeTest.downloadAndUnzipVSCode(vscodeVersion).then((vscodePath) => {
-		if (vscodePath) {
-			vscodeTest.runTests({
-				vscodeExecutablePath: vscodePath,
-				extensionDevelopmentPath: workspace,
-				extensionTestsPath: extensionTestsPath,
-				launchArgs: args
-			}).then(() => done()).catch((error) => {
-				console.log(`stdout: ${process.stdout}`);
-				console.log(`stderr: ${process.stderr}`);
-				console.error(`exec error: ${error}`);
-				done(error);
-			});
-		}
-	})
+	let vscodePath = await vscodeTest.downloadAndUnzipVSCode(vscodeVersion);
```

Comment:
We should probably always be downloading the latest stable to verify we're working against that.

Reply:
We download the same version as in the package.json. We should probably update the vscode engine in a separate PR.

Reply:
No, because that's the minimum version we support. That doesn't (and shouldn't) need to be the latest version so that users who don't want to/can't update VS Code for some reason can still use our extension if they want.

Reply:
But for testing we should just focus on the latest release - we could run tests against older versions if we wanted but generally that's not worth the extra time and effort.

Reply:
@Charles-Gagnon I tried updating it to latest and some tests are failing (non functional ones with no side-effects). I didn't get much time to investigate since I'm working on other tasks, but this can be added as another task for later. I'd like to get the tests running in the pipelines initially. Thoughts @kburtram ?

Reply:
This seems like a good task for the DRI to take up